drag-sort-listview-gradle/build.gradle

```diff
@@ -0,0 +1,26 @@
+apply plugin: 'com.android.library'
+
+// This file is here (rather than in the drag-sort-listview project)
+// so we can control how it builds without having to submit pull requests.
+
+dependencies {
+    compile 'com.android.support:support-v4:19.1.0'
+}
+
+android {
+    compileSdkVersion 7
```

Comment:
Ah yeah. You have to make sure the **Obsolete** option is checked (or **Show Package Details** in Android Studio) in order to see the SDKs that are that old. I'd also be happy to switch it to a more recent SDK if you want, especially if we're planning on dropping support for that :)

Or you if you can change it before merging then go for it! (I'm honestly not 100% familiar with how Github's pull requests work in terms of merging them in)